server/src/routers/christmasTreeAdmin.es6

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const express = require('express');
+
+const christmasTreeController = require('../controllers/christmas-tree.es6');
+
+const router = express.Router();
+
+/*GET christmas tree permit*/
+router.get('/christmas-trees/permits/:forestId/:startDate/:endDate', christmasTreeController.getPermits);
```

Comment:
again, let's make sure this endpoint is reflected in swagger.

Reply:
gotcha